sale_project_service_tracking_copy_tasks/views/sale_order_views.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Moduon Team S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+    <record model="ir.ui.view" id="view_order_form_inherit_sale_project">
+        <field name="model">sale.order</field>
```

Comment:
```suggestion
        <field name="model">Support for project with copied tasks</field>
```

Reply:
I think you assumed it was the name, not the model.
I have added missing name.